howto/troubleshoot/landing.md

```diff
@@ -1,11 +1,31 @@
 The topics in this guide describe some commonly encountered problems with Anbox Cloud and provide instructions for resolving them. If you encounter an issue with Anbox Cloud, check if any of the following scenarios help in resolving your issue.
 
+[note type="information" status="Note"]If the deployment is older than 3 months, you must upgrade Anbox Cloud to the latest version and see if the required fixes are already part of the upgrade. See [How to upgrade Anbox Cloud](https://discourse.ubuntu.com/t/how-to-upgrade-anbox-cloud/17750) for upgrade instructions.
```

Comment:
So, just a random thought:

What happens if we change the title of the page 'How to upgrade Anbox Cloud'? Will the link change on the discourse side?
e.g., if we change it to 'How to upgrade', will the link become `https://discourse.ubuntu.com/t/how-to-upgrade/17750`?

If the link does change, doesn't that mean that the previous link will become invalid? Should we remove title pages from link and only keep the thread id? (i.e., `https://discourse.ubuntu.com/t/17750`)

What do you think?

Reply:
The link does not change if you change the title. It just takes the title of the post when you create it the first time.

Reply:
Alright, thanks!